sig-scalability/README.md

```diff
@@ -12,11 +12,6 @@ SIG Scalability is responsible for defining and driving scalability goals for Ku
 
 The [charter](charter.md) defines the scope and governance of the Scalability Special Interest Group.
 
-## Meetings
```

Comment:
Why do you remove it? Meetings are happening biweekly (we just don't have recordings - hopefully the next one will be recorded). Agenda also started to be up-to-date in the last couple months.